app/src/data_loader/trade_loader/browser.py

```diff
@@ -0,0 +1,15 @@
+import asyncio
+import datetime
+import random
+import websockets
+
+async def time(websocket, path):
+    while True:
+        now = datetime.datetime.utcnow().isoformat() + "Z"
+        await websocket.send(now)
+        await asyncio.sleep(random.random() * 3)
```

Comment:
What is the purpose of this randomness and crossing by 3? Why do you sleep? 
Please change the name of function. time is an important module in python, we might face some conflicts.

Reply:
This file was for test purposes and is redundant. I should delete it.